ethereal-nexus-site/src/components/layout/header.astro

```diff
@@ -18,9 +18,9 @@ const { className } = Astro.props;
 >
   <div class="relative container flex items-center justify-between w-full">
     <div class="flex items-center gap-6 md:gap-8 lg:gap-10">
-      <a href="/" class="hidden items-center space-x-2 md:flex">
```

Comment:
What's the goal of removing this href?

Reply:
is only for showing the logo image not clickable. Need to check if this is ok to not have the logo image clickable in the documentation . If yes, then can remove the "a" and add another element to replace